frame/collective/src/lib.rs

```diff
@@ -78,26 +78,28 @@ pub const MAX_MEMBERS: MemberCount = 100;
 
 pub trait WeightInfo {
 	fn set_members(m: u32, n: u32, p: u32, ) -> Weight;
-	fn execute(m: u32, b: u32, ) -> Weight;
-	fn propose_execute(m: u32, b: u32, ) -> Weight;
-	fn propose_proposed(m: u32, p: u32, b: u32, ) -> Weight;
+	fn execute(b: u32, m: u32, ) -> Weight;
+	fn propose_execute(b: u32, m: u32, ) -> Weight;
+	fn propose_proposed(b: u32, m: u32, p: u32, ) -> Weight;
 	fn vote(m: u32, ) -> Weight;
-	fn close_early_disapproved(m: u32, p: u32, b: u32, ) -> Weight;
-	fn close_early_approved(m: u32, p: u32, b: u32, ) -> Weight;
-	fn close_disapproved(m: u32, p: u32, b: u32, ) -> Weight;
-	fn close_approved(m: u32, p: u32, b: u32, ) -> Weight;
+	fn close_early_disapproved(m: u32, p: u32, ) -> Weight;
+	fn close_early_approved(b: u32, m: u32, p: u32, ) -> Weight;
+	fn close_disapproved(m: u32, p: u32, ) -> Weight;
+	fn close_approved(b: u32, m: u32, p: u32, ) -> Weight;
+	fn disapprove_proposal(p: u32, ) -> Weight;
 }
 
 impl WeightInfo for () {
```

Comment:
same as asked by shawn in democracy https://github.com/paritytech/substrate/pull/6783/files better to default to the substrate runtime weight. it avoid breaking change, still not accurate weight in general case but slightly better.

Reply:
so I'll also remove them

Reply:
I think what was agreed was to implement it with a non breaking change default_weight (i.e. current polkadot weights)
Thus you should just copy the file from node/runtime/weight to here and modify it a bit so it implements for (). You can see democracy in master to take example, Shawn approved it.

Reply:
democracy on master does not provide a `()` implementation AFAICT?

Reply:
it is done here https://github.com/paritytech/substrate/blob/master/frame/democracy/src/default_weight.rs